app/javascript/packs/modal.js

```diff
@@ -0,0 +1,36 @@
+document.addEventListener('DOMContentLoaded', function(){
+  var buttons = document.querySelectorAll(".btn");
+  var closes = document.querySelectorAll(".modal-close");
+  
+  function openModal(event) {
+    var modal = document.querySelector("div[id="+event.currentTarget.id+"]");
+    modal.classList.add('opacity-100');
+    modal.classList.add('z-50');
+  }
```

Comment:
what do you think about adding a class name like "modal-open" and add their styles from CSS, just change the class name here on JS

Reply:
We are using the TailwindCSS tool, that's why we put it that way, since we are just learning to handle its classes.

Reply:
As @YilKanda said, we are using TailwindCSS to do not waste a lot of time styling the front end of the app since we have to focus a little bit more in the API.
He could use the **@apply** directive of Tailwind to encapsulate both _opacity-100_ and _z-50_ in a _modal-open_ class, however using the **@apply** directive for just two classes or styles is not recommend.